### DISCUSSION TITLE
Web UI crashed after inpainting mesh rendered successfully.

### DISCUSSION THREAD

**Discussion by deniznode** [thygate/stable-diffusion-webui-depthmap-script#189] [Q&A] [answered]
m2 8g mac, cpu:
in most cases the 3d mesh rendering cause web ui crash, the rendering seems always done in the backend tho. Where can I retrieve the obj file other than downloading from web ui? seems only one out of five attempts can I successfully download from ui after rendering. Wondering if there is way I can retrieve the rendered obj file other than downloading from web ui?

**Comment by thygate** [comment, marked as answer]
Yes, I've ran into problems too when using the gradio.model3d view, it does not seem to be very stable or consistent even. It also re transfers the last meshfile again and again when other components change ..

For now, please disable the 3d view in depthmap settings, I might remove it completely (3dview) in a next version ..

The files are saved in the same directory as the depthmaps, so output/extras-images for single images in the depth tab.

edit: the mesh is generated on the backend, but rendered in the browser by the gradio.model3d component.

**Reply by deniznode** [reply]
Thanks a lot! solved perfectly. cheers
